Arelle/Arelle
# EDGAR 23.4 preview

#### Reason for change
EDGAR release 23.4 preview

#### Description of change
See https://www.sec.gov/edgar/filermanual

#### Steps to Test
See https://www.sec.gov/structureddata/osdinteractivedatatestsuite

To get GUI of EdgarRenderer to launch in ixviewer-plus mode or production 23.2 mode, see lines to uncomment in plugin/EdgarRenderer/resources/Summarize.xslt (uncomment line 19 or 22)

**review**:
@Arelle/arelle
